packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -311,7 +314,7 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
           FieldMetadataEntity,
         );
 
-      const [existingFieldMetadata] = await fieldMetadataRepository.find({
+      const existingFieldMetadata = await fieldMetadataRepository.findOne({
```

Comment:
nope this is not equivalent :p typeORM makes two queries for a findOne and one for find!